tests/Database/DatabaseQueryBuilderTest.php

```diff
@@ -1376,6 +1376,42 @@ public function testOrWhereAny()
         $this->assertEquals(['%Taylor%', '%Otwell%', '%Otwell%'], $builder->getBindings());
     }
 
+    public function testWhereNone()
+    {
+        $builder = $this->getBuilder();
+        $builder->select('*')->from('users')->whereNone(['last_name', 'email'], 'like', '%Otwell%');
+        $this->assertSame('select * from "users" where not ("last_name" like ? or "email" like ?)', $builder->toSql());
+        $this->assertEquals(['%Otwell%', '%Otwell%'], $builder->getBindings());
+
+        $builder = $this->getBuilder();
+        $builder->select('*')->from('users')->whereNone(['last_name', 'email'], '%Otwell%');
```

Comment:
@einar-hansen you can remove `%` from value clarity (I think that you missed like operator in that case)

Reply:
I'm making a PR to fix this, thanks for the suggestion @siarheipashkevich